src/chromium/interoperability-with-cpp/using-cxx-in-chromium.md

```diff
@@ -0,0 +1,41 @@
+## Using cxx in Chromium
+
+In Chromium, we define an independent `#[cxx::bridge] mod` for each leaf-node
+where we want to use Rust. You'd typically have one for each
+`rust_static_library`. Just add
+
+```gn
+cxx_bindings = [ "my_rust_file.rs" ]
+   # just list files containing #[cxx::bridge]
```

Comment:
nit: s/ just list files / list of files /

Reply:
Done (with a tweak to make my intended meaning clearer - `list of files containing #[cxx::bridge], not all source files`